src/AppBundle/Mailjet/Message/NewsletterInvitationMessage.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace AppBundle\Mailjet\Message;
+
+use AppBundle\Entity\NewsletterInvite;
+
+final class NewsletterInvitationMessage extends MailjetMessage
+{
+    public static function createFromInvite(NewsletterInvite $invite, string $subscribeUrl): self
+    {
+        return new static(
+            $invite->getUuid(),
```

Comment:
Many mails will have the same uuid, is it ok? I've done it since this is done elsewhere but I'm not sure about this.